site/content/architecture/adr-001-use-adrs.mdx

```diff
@@ -0,0 +1,73 @@
+---
+type: ADR
+id: adr-001
+status: Accepted
+scope: global
+deciders: github group
+date: 2021-06-01
+title: Use ADRs
+---
+
+{/* Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. */}
+
+### Context
+
+ComposeUI has several very explicit goals that make the practice and
```

Comment:
this is talking about Compose UI. We probably want to update that before merging?

Reply:
Yes, you would want to replace the sample ADR with the ones for this project.

Reply:
If we have none should this be deleted?

Reply:
I realize you might be hesitant about the ADR but this documentation won't publish until we point GH pages to it. If you want to add documentation to this PR that is fine, but I intended for this to be a template that your team would fill out with documentation after initial merge. Once its ready, it would be published.